tick/optim/solver/tests/svrg_test.py

```diff
@@ -54,8 +85,105 @@ def test_variance_reduction_setting(self):
         self.assertEqual(svrg._solver.get_variance_reduction(),
                          _SVRG.VarianceReductionMethod_Last)
 
-        with self.assertRaises(ValueError):
-            svrg.variance_reduction = 'wrong_name'
+        msg = '^variance_reduction should be one of "avg, last, rand", ' \
+              'got "truc"$'
```

Comment:
Oh no, now everybody will know that we're French :)

Reply:
Replaced truc by stuff